sdk/src/Core/Amazon.Runtime/Credentials/RefreshingAWSCredentials.cs

```diff
@@ -28,6 +28,19 @@ public abstract class RefreshingAWSCredentials : AWSCredentials, IDisposable
     {
         private Logger _logger = Logger.GetLogger(typeof(RefreshingAWSCredentials));
 
+        public override DateTime? Expiration
```

Comment:
I would like to see docs on public properties as we add new ones at the minimum.

Reply:
I added an `<inheritdoc>` tag so the documentation from the base class is propagated.